Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -323,3 +323,17 @@ private extension WorldScaleGeoTrackingSceneView {
         )
     }
 }
+
+public extension WorldScaleGeoTrackingSceneView {
```

Comment:
This is not the right place for the extension. Should be on `ARSwiftUIViewProxy`